### PR TITLE
[DROOLS-6094] Fix package name declaration from files containing dash in the name 

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @mariofusco @jiripetrlik

see https://issues.redhat.com/browse/DROOLS-6094 https://issues.redhat.com/browse/RHPAM-3521

### CODE REVIEW

**Review by jiripetrlik — APPROVED** [review]
@gitgabrio 
Thank you for quick fix. Looks good to me!

File: kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/utils/KiePMMLModelUtils.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ private KiePMMLModelUtils() {
      * @return
      */
     public static String getSanitizedPackageName(String modelName) {
-        return modelName.replace(" ", "").replace("_", "").toLowerCase();
+        return modelName.replace(" ", "").replace("-", "").replace("_", "").toLowerCase();
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/utils/KiePMMLModelUtils.java:30]: What about "clone" the same logic of this method?
https://github.com/kiegroup/drools/blob/master/kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/codegen/feel11/CodegenStringUtil.java#L31-L50

**Review by danielezonca — APPROVED** [review]
Thanks for the PR, approving with minor suggestion

File: kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/utils/KiePMMLModelUtils.java (hunk 1 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-commons/src/main/java/org/kie/pmml/commons/utils/KiePMMLModelUtils.java:30]: I prefer to not have "_" or similar in packagename/classname. And the logic of that method is different of what is needed here

**Pull request merged**